IA/IA_upload.py

```diff
@@ -0,0 +1,107 @@
+import os
+import argparse
+import requests
+import boto
+from boto.s3.connection import OrdinaryCallingFormat
+from boto.s3.multipart import MultiPartUpload
+import asyncio
+import xmltodict
+from io import BytesIO
+from settings import (
+    CHUNK_SIZE,
+    OSF_COLLECTION_NAME,
+    IA_ACCESS_KEY,
+    IA_SECRET_KEY,
+)
+
+HERE = os.path.dirname(os.path.abspath(__file__))
+
+
+class IAException(Exception):
+    pass
```

Comment:
Nothing really happens here. How are we supposed to know if a file failed to upload??

Reply:
replaced this with requests builtin exception